pkg/storage/test/tuples.go

```diff
@@ -41,7 +41,7 @@ func ReadChangesTest(t *testing.T, datastore storage.OpenFGADatastore) {
 		err := datastore.Write(ctx, storeID, nil, []*openfgav1.TupleKey{tk1, tk2})
 		require.NoError(t, err)
 
-		changes, continuationToken, err := datastore.ReadChanges(ctx, storeID, "", storage.PaginationOptions{PageSize: 1}, 0)
+		changes, continuationToken, err := datastore.ReadChanges(ctx, storeID, "", storage.NewPaginationOptions(1, ""), 0)
```

Comment:
Like we discussed offline, can you ammend the godoc here https://github.com/openfga/openfga/blob/883fb6c63bea84778035cac383d2baf132237b42/pkg/storage/storage.go#L236-L248

to say that the implementation must return `storage.ErrNotFound` and no continuation token when there are no more changes left? (And any other details that you may have noticed during development)